docs/detections/machine-learning/machine-learning.asciidoc

```diff
@@ -76,6 +76,19 @@ prior to the time they are enabled. After jobs are enabled, they continuously
 analyze incoming data. When jobs are stopped and restarted within the two-week
 time frame, previously analyzed data is not processed again.
 
+[float]
+[[ml-integrations]]
+=== Jobs in advanced analytics (UEBA) Elastic integrations
+
+{ml} jobs can also be installed via https://docs.elastic.co/integrations[Elastic integrations]. We currently have the following Advanced Analytics integrations for Security:
+
+* https://docs.elastic.co/integrations/ded[Data Exfiltration Detection]
+* https://docs.elastic.co/integrations/dga[Domain Generation Algorithm Detection]
+* https://docs.elastic.co/integrations/lmd[Lateral Movement Detection]
+* https://docs.elastic.co/integrations/problemchild[Living off the Land Attack Detection]
+
+Read more about the {ml} jobs enabled by these Integrations in the https://www.elastic.co/guide/en/security/current/prebuilt-ml-jobs.html[Prebuilt jobs page].
```

Comment:
Will we change this link to [this](https://stack-docs_2496.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-ml-jobs.html#security-integrations-jobs) after the PR is merged?

Reply:
Good call, I can link to the sub-section